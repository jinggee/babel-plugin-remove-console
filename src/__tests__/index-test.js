import * as babel from '@babel/core';
import plugin from '../index';

const example = `
console.log('test');

function test() {
  console.log('call console in a method');
}
`;

it('remove console works with prod env', () => {
  const { code } = babel.transform(example,
    { plugins: [[plugin, { env: 'prod' }]] });
  expect(code).toMatchSnapshot();
  expect(code).toBe(`"use strict";

function test() {}`);
});
