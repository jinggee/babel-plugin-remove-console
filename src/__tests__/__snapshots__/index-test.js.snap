// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`remove console works 1`] = `
"\\"use strict\\";

function test() {}"
`;
